site/about/glossary/_developer-tools.qmd

```diff
@@ -0,0 +1,33 @@
+decorator, Python decorator
+: A [design pattern in Python](https://www.datacamp.com/tutorial/decorators-python) that allows a user to add new functionality to an existing object without modifying its structure.
+
+  - Decorators are a simpler way for users to run their
```

Comment:
GLOBAL COMMENT: Bulleted lists should have at least a couple of list items. If not, just combine with the previous text on line 2 or leave them as plain paragraphs. 

(There's a few of these here and there all throughout.)

Reply:
Can I make this edit in a future PR 😅

Reply:
Sure — my suggestion with any deferred comment would be to capture it in a Shortcut story.

Reply:
FYI

```
8 results - 6 files

site/about/glossary/_models.qmd:
  11  model inventory
  12: : A systematic and organized record of all quantitative and qualitative models used within an organization. This inventory facilitates oversight, tracking, and assessment by listing each model's purpose, characteristics, owners, validation status, and associated risks. 
  13: 
  14:     - Also see: [ValidMind model inventory](../../about/glossary.html#platform-model-inventory)
  15  

  19  model risk
  20: : The potential for financial loss, incorrect decisions, or unintended consequences resulting from errors or inaccuracies in AI or machine learning models. Model risk typically arises from incorrect or inappropriate use of models, inaccurate assumptions, or limitations in data quality. 
  21: 
  22:     - e.g. Consequences of unmitigated model risk can include adverse outcomes such as financial loss, damage to reputation, and regulatory penalties.
  23  

site/about/glossary/_mrm.qmd:
  32  model validator
  33: : responsible for conducting independent assessments of models to ensure their accuracy, reliability, and appropriateness for intended purposes. The role involves evaluating a model's conceptual soundness, data integrity, calibration methods, and overall performance, typically using out-of-sample datasets. 
  34: 
  35:     - Model validators identify potential risks and weaknesses, ensuring that models within an organization meet established standards and regulatory requirements, and provide recommendations to model developers for improvements or modifications.
  36  
  37  three lines of defense
  38: : A structured approach to model risk management, consisting of three independent functions. 
  39: 
  40:     - The first line consists of business units responsible for model development, validation, and implementation. They ensure that models are accurate, robust, and fit for purpose. 
  41      - The second line is an independent model risk oversight function that provides a governance framework and guidance for model risk management.

site/about/glossary/key_concepts/_metrics.qmd:
  1  metrics, custom metrics
  2: : Metrics are a subset of tests that do not have thresholds. Custom metrics are functions that you define to evaluate your model or dataset. These functions can be registered with ValidMind to be used in the platform.
  3: 
  4:     -  In the context of ValidMind's Jupyter Notebooks, metrics and tests can be thought of as interchangeable concepts. 

site/about/glossary/key_concepts/_report.qmd:
  1  validation report
  2: : A formal document produced after a model validation process, outlining the findings, assessments, and recommendations related to a specific model's performance, appropriateness, and limitations. Provides a comprehensive review of the model's conceptual framework, data sources and integrity, calibration methods, and performance outcomes. 
  3: 
  4:     - Within model risk management, the validation report is crucial for ensuring transparency, demonstrating regulatory compliance, and offering actionable insights for model refinement or adjustments.

site/about/glossary/key_concepts/_template.qmd:
  1  template, documentation template
  2: : Functions as a test suite and lays out the structure of model documentation, segmented into various sections and sub-sections. Documentation templates define the structure of your model documentation, specifying the tests that should be run, and how the results should be displayed.
  3: 
  4:     - ValidMind templates come with pre-defined sections, similar to test placeholders, including boilerplates and spaces designated for documentation and test results. When rendered, produces a document that model developers can use for model validation.

site/about/glossary/key_concepts/_test.qmd:
  1  test
  2: : A function contained in the {{< var vm_dev >}}, designed to run a specific quantitative test on the dataset or model. Test results are sent to the ValidMind platform to generate the model documentation according to the template that is associated with the documentation.
  3: 
  4:     - Tests are the building blocks of ValidMind, used to evaluate and document models and datasets, and can be run individually or as part of a suite defined by your model documentation template.
```

Reply:
Addressed via 114c458232b02f582e6a0e73adcfb5586733b403.